PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Forget RADIUS WiFi Network.md

```diff
@@ -0,0 +1,141 @@
+#### Name
+
+Windows - Forget RADIUS WiFi Network | v1.0 JCCG
+
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+# Enter the SSID of the Radius network
+$RadiusSSID = ''
+
+# Change Overwrite to $true if you would like to re-create existing Scheduled Tasks
+# This should only be used if redistributing this command to devices
+$Overwrite = $false
+
+
+#### DO NOT EDIT BELOW THIS LINE ####
+$removeWifips1 = @"
+function Get-WifiProfile {
+    [cmdletbinding()]
+    param
+    (
+         [System.Array]`$Name = `$NULL
+    )
+    Begin {
+        `$list = ((netsh.exe wlan show profiles) -match '\s{2,}:\s') -replace '.*:\s' , ''
+        `$ProfileList = `$List | Foreach-object { [pscustomobject]@{Name = $_ } }
```

Comment:
Missing `

```suggestion
        `$ProfileList = `$List | Foreach-object { [pscustomobject]@{Name = `$_ } }
```
<img width="707" alt="Screenshot 2024-10-15 at 1 02 09 PM" src="https://github.com/user-attachments/assets/bf7f767e-9306-4a86-b6ec-bdcc37b730f7">

Reply:
Fixed this